src/Course.h

```diff
@@ -17,8 +17,16 @@ struct Game;
 
 const int MAX_EDIT_COURSE_TITLE_LENGTH = 16;
 
-inline PlayMode CourseTypeToPlayMode( CourseType ct ) { return (PlayMode)(PLAY_MODE_NONSTOP+ct); }
-inline CourseType PlayModeToCourseType( PlayMode pm ) { return (CourseType)(pm-PLAY_MODE_NONSTOP); }
```

Comment:
I can't recall, but indeed it is not used. The previous implementation likely has unanticipated behavior when calling this function with certain `PlayMode`s because there are more `PlayMode`s than `CourseType`s. There's not a one-to-one mapping, like this function seems to imply.